source/assets/css/components/_navigation.scss

```diff
@@ -8,3 +8,21 @@
     height: 100vh;
   }
 }
+
+.skip-link {
+  display: block;
+  padding: 0.25em 1em;
+  transform: translateY(calc(-100% - 1em));
+  position: absolute;
+  left: 1em;
+  top: 1em;
+  z-index: 9999;
+
+  &:focus {
+    transform: translateY(0);
+    transition: all .1s ease-in-out;
```

Comment:
Could this transition be on the parent instead, so that the link's exit is also animated?